### PR TITLE
Improved description when the stack not export port. Add link for registry catalog name. Update info of author zhangsean.

### PR DESCRIPTION
**Pull request by zhangsean**
1. Improved description when the stack not export port. 
2. Add link for registry catalog name. 
3. Update info of author zhangsean.

### CODE REVIEW

**Comment by upccup** [comment]
ping @my9074

File: frontend/src/registry/list/catalog.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
         <md-card>
             <md-card-title>
                 <md-card-title-text>
-                    <span class="md-headline">{/catalog.Name/}</span>
+                    <a class="md-headline registry-link" data-ui-sref="registry.updateCatalog({catalog_id: catalog.Id})" href="/registry/updateCatalog/{/catalog.Id/}">{/catalog.Name/}</a>
```
Comment (my9074) [inline frontend/src/registry/list/catalog.html]: href attribute don't need it, ui-sref instead of it.

**Comment by vitan** [comment]
retest this please

**Comment by vitan** [comment]
retest this please

**Comment by vitan** [comment]
retest this please

**Pull request merged**